tools/changelog.py

```diff
@@ -76,9 +76,18 @@ def normalize_title(title):
     See the test suite for the potential matches.
     GitHub sends us a unicode string.
     """
-    if '🚀' in title:
-        title = title.replace('🚀', '')
-        title = title.strip()
+    # if '🚀' in title:
+    #     title = title.replace('🚀', '')
+    #     title = title.strip()
+    #     title = 'NPM update - {title}.'.format(title=title)
+
+    if 'Bump' in title:
+        """
+        index [0] -> Bump always for dependabot
+        index [1:] -> name of package and version info
+        """
+        title = title.split()[1:]
+        title = ' '.join(title)
```

Comment:
so there are a couple of ways to do that. You found one. Here's another one using [partition](https://docs.python.org/3/library/stdtypes.html#str.partition)

```python
>>> # the current code
>>> title = 'Bump grunt from 1.0.4 to 1.1.0'
>>> ' '.join(title.split()[1:])
'grunt from 1.0.4 to 1.1.0'
>>> # another solution using partition
>>> title = 'Bump grunt from 1.0.4 to 1.1.0'
# partition returns a tuple with the 3 parts of the left partition
>>> title.partition(' ')
('Bump', ' ', 'grunt from 1.0.4 to 1.1.0')
>>> # so straightforward this will return what we want in one line.
>>> title.partition(' ')[2]
'grunt from 1.0.4 to 1.1.0'
```

@apsychogirl but maybe we don't need to remove the `bump` word depending on what you decide for the test in our previous comments

Reply:
> so there are a couple of ways to do that. You found one. Here's another one using [partition](https://docs.python.org/3/library/stdtypes.html#str.partition)
> 
> ```python
> >>> # the current code
> >>> title = 'Bump grunt from 1.0.4 to 1.1.0'
> >>> ' '.join(title.split()[1:])
> 'grunt from 1.0.4 to 1.1.0'
> >>> # another solution using partition
> >>> title = 'Bump grunt from 1.0.4 to 1.1.0'
> # partition returns a tuple with the 3 parts of the left partition
> >>> title.partition(' ')
> ('Bump', ' ', 'grunt from 1.0.4 to 1.1.0')
> >>> # so straightforward this will return what we want in one line.
> >>> title.partition(' ')[2]
> 'grunt from 1.0.4 to 1.1.0'
> ```
> 
> @apsychogirl but maybe we don't need to remove the `bump` word depending on what you decide for the test in our previous comments

How about if we follow the same trend as already used in changelog.md

`>>> title="Bump grunt from 1.0.4 to 1.1.0"
>>> title=title.partition('grunt')
>>> title = 'NPM update - Update {title} to latest version.'.format(title=title[1])
>>> print(title)
NPM update - Update grunt to latest version.`

Reply:
1. note that this example used grunt, but it could be anything, next time it will not be grunt. :) `.partition('grunt')` will not work. 
2. If the keyword is already known and extracted, then `partition` is not needed at all. :)